GitHub-Test-3/logan-gitTest3.md

```diff
@@ -0,0 +1,112 @@
+# Logan's Mardown file for Git Exercise 3
+
+# The Lorax
+## By Dr. Seuss
```

Comment:
@luh429 That's a neat idea to encode The Lorax in a little bit of markdown. What I'm writing here is also in markdown syntax, in a **line comment**, which is a comment on a specific line of code on a commit to a remote repo. You'll be writing a line comment like this on another student's commit / Pull Request for the last stage of our git/GitHub test series.